README.md

```diff
@@ -1,15 +1,228 @@
+
+```
+--input /path/to/samples.csv
+```
+
+### 2. Prepare the reference materials 
+
+To run this pipeline you will need the following reference files:
+
+- Indexed reference genome in FASTA format. You will need to download and index a copy of the reference genome you would like to use. Reference FASTA files must be accompanied by a .fai index file. If you are working with a species that has a public reference genome, you can download FASTA files from the [Ensembl](https://asia.ensembl.org/info/data/ftp/index.html), [UCSC](https://genome.ucsc.edu/goldenPath/help/ftp.html), or [NCBI](https://www.ncbi.nlm.nih.gov/genome/doc/ftpfaq/) ftp sites. 
+- You can use our [IndexReferenceFasta-nf pipeline](https://github.com/Sydney-Informatics-Hub/IndexReferenceFasta-nf) to generate indexes. This pipeline uses the following tools for generating specific index files.
+
+|Tool    | Index file(s)   |
+|-------------------|:--------------------------------- |
+|samtools            | .fai                               |
+|bwa            | .amb, .ann, .bwt, .pac, .sa                               |
+|picard            | .dict                              |
+
+When you run the pipeline, you will use the mandatory `--ref` parameter to specify the location and name of the reference.fasta file: 
+```
+--ref /path/to/reference.fasta
+```
+***Note***: You must specify the full path for the reference fasta, even if it is in your working directory.
+
+- You will also need to download 
+
+### 3. Clone this repository 
+
+Download the code contained in this repository with: 
+
+```
+git clone https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf
+```
+
+This will create a directory with the following structure: 
+```
+Somatic-shortV-nf/
+├── LICENSE
+├── README.md
+├── config/
+├── images/
+├── scripts/
+├── main.nf
+├── modules/
+└── nextflow.config
+```
+The important features are: 
+
+* **main.nf** contains the main nextflow script that calls all the processes in the workflow.
+* **nextflow.config** contains default parameters to use in the pipeline.
+* **modules** contains individual process files for each step in the workflow. 
+* **config** contains infrastructure-specific config files (this is currently under development)
+
+### 4. Run the pipeline 
+
+The most basic run command for this pipeline is: 
+
+```
+nextflow run main.nf --input samples.csv --ref reference.fasta  --intervalList_path path_to_intervals --ponvcf pon.vcf.gz
+        
+```
+
+By default, this will generate `work` directory, `results` output directory and a `runInfo` run metrics directory inside the results directory. 
+
+To specify additional optional tool-specific parameters, see what flags are supported by running:
+
+```
+nextflow run main.nf --help 
+```
+
+If for any reason your workflow fails, you are able to resume the workflow from the last successful process with `-resume`. 
```

Comment:
Is this currently working? If not, then you need to clarify that it is currently not functional.

Reply:
I have tested the pipeline. It works fine.

The description in Readme.md should assist the user. Please give it a try.